src/ui/Card/Card.module.css

```diff
@@ -0,0 +1,35 @@
+.card {
+  color: var(--font-color-1);
+  border-radius: var(--radii-level-2);
+  background-color: var(--bg-color-1);
+  padding: 0;
+}
+
+.card.dark {
+  background-color: var(--brand-color-1);
+  color: var(--font-color-2);
+}
+
+.cardHeader {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 2rem 1.5rem 2rem;
+}
+
+.cardTitle {
+  font-weight: var(--font-weight-bold);
+  line-height: 1;
+  margin-bottom: var(--margin-level-3);
+}
```

Comment:
This is not needed right? We can adhere to earlier styles made with Heading component